configs/sites/tier1/aws-pcluster/README.md

```diff
@@ -167,6 +157,60 @@ mkswap /swapfile
 swapon /swapfile
 swapon -s
 echo "/swapfile swap swap defaults 0 0" >> /etc/fstab
+```
+
+
+
+2. The PCluster base image includes the Intel MPI library but does not include
+the Intel compiler toolchain. Installing the Intel compiler toolchain with apt
+shuffles the `/opt/intel` directory badly and places the libraries and tools in
+nonstandard locations due to conflicts with the installed MPI library. The
+following instructions are used to clear the existing Intel MPI and install a
+clean and unified Intel toolchain.
+
+```
+rm -rf /opt/intel
+rm -rf /var/intel
+
+mkdir -p /opt/intel/src
+pushd /opt/intel/src
+
+# Download Intel install assets.
+wget -O cpp-compiler.sh https://registrationcenter-download.intel.com/akdlm/IRC_NAS/d85fbeee-44ec-480a-ba2f-13831bac75f7/l_dpcpp-cpp-compiler_p_2023.2.3.12_offline.sh
+wget -O fortran-compiler.sh https://registrationcenter-download.intel.com/akdlm/IRC_NAS/0ceccee5-353c-4fd2-a0cc-0aecb7492f87/l_fortran-compiler_p_2023.2.3.13_offline.sh
+wget -O tbb.sh https://registrationcenter-download.intel.com/akdlm/IRC_NAS/c95cd995-586b-4688-b7e8-2d4485a1b5bf/l_tbb_oneapi_p_2021.10.0.49543_offline.sh
+wget -O mpi.sh https://registrationcenter-download.intel.com/akdlm/IRC_NAS/4f5871da-0533-4f62-b563-905edfb2e9b7/l_mpi_oneapi_p_2021.10.0.49374_offline.sh
+wget -O math.sh https://registrationcenter-download.intel.com/akdlm/IRC_NAS/adb8a02c-4ee7-4882-97d6-a524150da358/l_onemkl_p_2023.2.0.49497_offline.sh
+
+# Install the Intel assets.
+sh cpp-compiler.sh -a --silent --eula accept 2>&1 | tee install.cpp-compiler.log
+sh fortran-compiler.sh -a --silent --eula accept | tee install.fortran-compiler.log
+sh tbb.sh -a --silent --eula accept | tee install.tbb.log
+sh mpi.sh -a --silent --eula accept | tee install.mpi.log
+sh math.sh -a --silent --eula accept | tee install.math.log
+
+popd
+```
+
+
+3. Install docker
+```
+# Docker
+# See https://docs.docker.com/engine/install/ubuntu/
+apt-get update
+apt-get install ca-certificates curl gnupg lsb-release
+mkdir -m 0755 -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null
+apt-get update
+apt install -y docker-ce \
+    docker-ce-cli containerd.io \
+    docker-buildx-plugin \
+    docker-compose-plugin
+docker run hello-world
+# DH* TODO 2023/02/21: Add users to group docker so that non-root users can run it
```

Comment:
Did you want to address lines 212 and 213?

Reply:
Not this time - the cluster made with this config is a single-purpose cluster being used by Fabio. Actually I left this section in to maintain instructions but the current cluster doesn't need Docker so I skipped it this time.

Reply:
Makes sense. Let me know if you want me to merge, please

Reply:
@climbfuji - I'm out tomorrow, but once this is reviewed it will be ready to merge.

If Fabio's work uncovers issues with this config I'll just follow up with a new change. For now, spack-stack builds and jedi-bundle ctests pass so I have nothing to add.